templates/components/input.build.hbs

```diff
@@ -1,26 +1,15 @@
 <div v-if="description" class="help-block description">\{{ description }}</div>
-<template v-if="readonly">
-  <p class="form-control-static">\{{ value }}</p>
-  <input
-    type="hidden"
-    class="form-control"
-    v-model.trim.lazy="value"
-    v-on:change="updateValue()"
-    v-on:input="onInput($event)"
-    :name="name"
-    :id="name"
-    :placeholder="placeholder"
-  />
-</template>
-<input v-else
+
```

Comment:
Can you please remove new empty blank line.

Reply:
done changes